client/src/Clusters/TopCharts/TopCharts.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import SingleAppContainer from './SingleAppContainer.js';
+import {topFree} from './mockData.js';
+import {topGrossing} from './mockData.js';
+import {topPaid} from './mockData.js';
+import {trending} from './mockData.js';
```

Comment:
You can import this in one line

`import {topFree, topGrossing, topGrossing, trending} from './mockData.js';`